packages/surveysparrow_flutter_sdk/lib/surveysparrow.dart

```diff
@@ -0,0 +1,749 @@
+library surveysparrow;
+
+export 'package:surveysparrow_flutter_sdk/models/customSurveyTheme.dart';
+export 'package:surveysparrow_flutter_sdk/models/firstQuestionAnswer.dart';
+export 'package:surveysparrow_flutter_sdk/models/answer.dart';
+
+import 'package:flutter/material.dart';
+import 'package:http/http.dart' as http;
+import 'package:surveysparrow_flutter_sdk/components/common/bottomNavigation.dart';
+import 'package:surveysparrow_flutter_sdk/components/common/footer.dart';
+import 'package:surveysparrow_flutter_sdk/components/common/header.dart';
+import 'package:surveysparrow_flutter_sdk/components/loadingScreen.dart';
+import 'package:surveysparrow_flutter_sdk/components/thankyou.dart';
+import 'package:surveysparrow_flutter_sdk/components/welcome.dart';
+import 'package:surveysparrow_flutter_sdk/helpers/question.dart';
+import 'package:surveysparrow_flutter_sdk/logics/displayLogic.dart';
+import 'package:surveysparrow_flutter_sdk/logics/thankYou.dart';
+import 'package:surveysparrow_flutter_sdk/models/customSurveyTheme.dart';
+import 'package:surveysparrow_flutter_sdk/models/theme.dart';
+import 'dart:convert';
+import 'package:sizer/sizer.dart';
+import 'helpers/answers.dart';
+import 'dart:async';
+import 'package:url_launcher/url_launcher.dart';
+
+class SurveyModal extends StatelessWidget {
+  final String token;
+  final String domain;
+  final Map<String, String>? customParams;
+  final dynamic? firstQuestionAnswer;
+  final CustomSurveyTheme? euiTheme;
+  final Function? currentlyCollectedAnswers;
+  final Function? allCollectedAnswers;
+  final Map<dynamic, dynamic>? surveyData;
+  final Function? onSubmitCloseModalFunction;
```

Comment:
@sachin-kumar56, If possible can we keep some variables private and don't expose them to the extended classes?